### DISCUSSION TITLE
Toggle Combined Batch Images

### DISCUSSION THREAD

**Discussion by kklouzal** [Kilvoctu/aiyabot#197] [Ideas]
a /setting to turn off combining batched images into a single image, can force combined images again if the image size is greater than 10.

**Comment by Kilvoctu** [comment]
Heya, I'm probably not going to do this. Batch wasn't even in the original scope of the aiyabot, and I was ready to remove it completely until convinced otherwise.  
Then people bombarded me negative feedback on this and that while I was developing batch which is partly why I'm not so active on this project anymore, so I'd be happy to not touch it again 🤷‍♀️😅  

If someone wants to make a PR for it, I'll merge it.

**Reply by kklouzal** [reply]
I'll look into the codebase and see if there's anything I can do to contribute on this front. I appreciate you :)
I like setting up 4 images to return for a single prompt, usually I can get my desired output in 1 of those 4 images, so i'd have to be re-running the prompt a few times anyways, so batch just makes it a lot easier and faster.
